config/toggles.js

```diff
@@ -93,7 +93,7 @@ const longTermToggles = {
   requireHttps: environmentVariableTrue(pe, 'REQUIRE_HTTPS'),
 
   //Enable Rooms functionality
```

Comment:
I'm going to be a stickler for all the style details because as an open source project, consistency is important.